#!/usr/bin/env python
PACKAGE = "unstruct_navigation_ros"
import roslib; roslib.load_manifest(PACKAGE)
from math import pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("logging_vehicle_states", bool_t, 0, "logging_vehicle_states", False) ## used with data generation script
gen.add("clear_buffer", bool_t, 0, "Clear logging buffer", True) ## used with data generation script
gen.add("output_image_mode", int_t, 0, "output_image_mode", 0, 0, 10) 
gen.add("paths_pub_enable", int_t, 0, "output_image_mode", 0, 0, 1) 

# gen.add("dist_heuristic_cost_scale", double_t, 0, "dist_heuristic_cost_scale", 3.0, 0, 100.0) 
# gen.add("rollover_cost_scale", double_t, 0, "rollover_cost_scale", 0.0, 0, 100.0)
# gen.add("model_error_weight", double_t, 0, "model_error_weight", 0.0, 0, 100.0)
# gen.add("local_map_cost_weight", double_t, 0, "local_map_cost_weight", 5.0, 0, 100.0)
# gen.add("error_std_scale", double_t, 0, "error_std_scale", 0.64, 0, 1.0)
# gen.add("error_mean_scale", double_t, 0, "error_mean_scale", 0.12, 0, 1.0)



exit(gen.generate(PACKAGE, "unstruct_navigation_ros", "dyn"))